draft-tschofenig-rats-psa-token.md

```diff
@@ -865,35 +865,68 @@ Content-Formats" registry {{IANA-CoAP-Content-Formats}}:
 
 --- back
 
-# Example
+# Examples
 
-The following example shows a PSA attestation token for an hypothetical system
-comprising two measured software components (a boot loader and a trusted RTOS).
-The attesting device is in a lifecycle state {{sec-security-lifecycle}} of
+The following examples show PSA attestation tokens for an hypothetical system
+comprising a single measured software component.
+The attesting device is in a lifecycle state ({{sec-security-lifecycle}}) of
 SECURED.  The attestation has been requested from a client residing in the
-SPE:
+SPE.
+
+The example in {{ex-sign1}} illustrates the case where the IAK is an asymmetric key.  A COSE Sign1 envelope is used to wrap the PSA claims-set.
+
+{{ex-mac0}} illustrates the case where the IAK is a symmetric key (TF-M is compiled with `SYMMETRIC_INITIAL_ATTESTATION`) and a COSE Mac0 envelope is used instead.
```

Comment:
I would omit the note about how TF-M is compiled.

Reply:
done in [9101b87](https://github.com/thomas-fossati/draft-psa-token/pull/102/commits/9101b8741005d78679e9c43fc74857fba92c80b5)